lib/temporal.rb

```diff
@@ -160,6 +211,8 @@ def metrics
 
     private
 
+    include Concerns::Payloads
```

Comment:
In Ruby we normally place includes at the top of the class definition. The scope of the included method is defined by the module that is being included, not where you include it